src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog.js

```diff
@@ -185,6 +185,36 @@ const TwoWindingsTransformerModificationDialog = ({
         }
     };
 
+    const isRatioTapChangerEnabled = useCallback(
+        (twtEditData) => {
+            const ratioTapEnabledInEditData =
```

Comment:
Is it possible to remove this one and test only `ratioTapFormHasBeenEdited || ratioTapEnabledInTwtToModify` ?
I don't see how ratioTapEnabledInEditData would give a different result than ratioTapFormHasBeenEdited since ratioTapFormHasBeenEdited is more broad

Reply:
we can have the enabled value set to false then we don't have to check if there's modifications or not in the form